procedures/MT/rotator/rotator_performance.ipynb

```diff
@@ -0,0 +1,655 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        """Track the target with a lasting time.
</pre><p>lasting_time is the duration? I've not heard "lasting_time before". Might be worth adding a definition of "lasting time"</p><p> </p><p>Also, <code>this value should be &gt;= 0</code>  --&gt; replace should with must.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Renamed in the commit: [30d19d5](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/30d19d514c64981689c466b89131d18ff78a1464). Thanks!

Reply:
<p>Thanks. </p><p>Just note you missed the "lasting time" in the description.</p>

Reply:
Fixed in commit: [da2822c](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/da2822c02b133ed099461c12c208f882ed94bd6d). Thanks!